core/docs/spark-qualification-tool.md

```diff
@@ -32,50 +20,70 @@ applicable environments.  Here are the cluster information for the ETL benchmark
+   - [HTML Report](#html-report)
+- [Example Reports and File Output](#example-reports-and-file-output)
+- [Compiling the Tools Jar from Source](#compiling-the-tools-jar-from-source)
+- [S3 File Path Support and Configuration](#s3-file-path-support-and-configuration)
+- [Tools Jar Maven Dependency](#qualification-tool-maven-dependency-snippet)
 
-This document covers below topics:
 
-* TOC
-{:toc}
 
-## How to use the Qualification tool
+## Running the Qualification Tool
 
-The Qualification tool can be run in three different ways. One is to run it as a standalone tool on the
-Spark event logs after the application(s) have run, the second is to be integrated into a running Spark
-application using explicit API calls, and the third is to install a Spark listener which can output
-results on a per SQL query basis.
+The Qualification tool can be run three different ways:
+1. A standalone application running on Spark event logs after the applications are done
+2. Integrated into a running Spark application using explicit API calls
+3. Installing a Spark listener to output results on a per SQL query basis
 
-In running the qualification tool standalone on Spark event logs, the tool can be run as a user tool command
-via a [pip package](https://pypi.org/project/spark-rapids-user-tools/) for CSP environments (Google Dataproc,
-AWS EMR, Databricks AWS) or as a java application for other environments.
+Standalone mode can be run as a java application or using the `spark_rapids_user_tools` application that is available as the `spark-rapids-user-tools` [pip package](https://pypi.org/project/spark-rapids-user-tools/).
 
-## Running the Qualification tool standalone for CSP environments on Spark event logs
-### User Tools Prerequisites and Setup for CSP environments
+## Running the Standalone Qualification Tool for CSP Environments on Spark Event Logs
+### User Tools Prerequisites and Setup for CSP Environments
 
-* [Dataproc](https://github.com/NVIDIA/spark-rapids-tools/blob/main/user_tools/docs/user-tools-dataproc.md)
-* [EMR](https://github.com/NVIDIA/spark-rapids-tools/blob/main/user_tools/docs/user-tools-aws-emr.md)
-* [Databricks AWS](https://github.com/NVIDIA/spark-rapids-tools/blob/main/user_tools/docs/user-tools-databricks-aws.md)
+* [Dataproc](../../user_tools/docs/user-tools-dataproc.md)
+* [EMR](../../user_tools/docs/user-tools-aws-emr.md)
+* [Databricks AWS](../../user_tools/docs/user-tools-databricks-aws.md)
 
 ### Qualify CPU Workloads for Potential Cost Savings and Acceleration with GPUs
 
-The qualification tool will run against logs from your CSP environment and then will output the applications
-recommended for acceleration along with estimated speed-up and cost saving metrics.
+The currently supported CSPs are *dataproc*, *emr*, and *databricks-aws*. The Qualification tool will run against logs from the specific CSP environment and will output GPU acceleration recomendations with estimated speed-up and cost saving metrics.
```

Comment:
We probably should reference the user-tools documentation to avoid conflicting information.
`databricks-azure` is added recently.